.github/workflows/test_tarball.yml

```diff
@@ -0,0 +1,216 @@
+        name: test-suite.log
+        path: build/test-suite.log
+# Start building tarball
+    - name: Install pandoc
+      uses: nikeee/setup-pandoc@v1
+    - name: Test pandoc
+      run: pandoc --version
+# Build the tarball
+    - name: Make tarball
+      run: cd build && make dist && mkdir tarballs && mv *.tar.gz tarballs
+# Upload the tarball
+    - name: upload tarball
+      uses: actions/upload-artifact@v4
+      with:
+        name: tarballs
+        path: build/tarballs
+ # Perform a make distcheck to check the distribution locally
+ #   - name: make distcheck
+ #     run: |
+ #           cd build
+ #           make distcheck
+
+
+  test-tarball:
+    needs: build
+    runs-on: ubuntu-20.04
+    container: dlramr/t8code-ubuntu:t8-dependencies
+    timeout-minutes: 90
+    steps: 
+    #
+    # Setup and bootstrap
+    #
+    - uses: actions/checkout@v4
+      with: 
+        fetch-depth: 0
+    - name: Update packages
+      run: apt-get update && apt-get upgrade -y
+    # This seems to be necessary because of the docker container
+    - name: disable ownership checks
+      run: git config --global --add safe.directory '*'
+    - name: init submodules
+      run: git submodule init
+    - name: update submodules
+      run: git submodule update
+    - name: bootstrap
+      run: ./bootstrap
+
+
+    - name: Download tarball
+      uses: actions/download-artifact@v4
+      with:
+          name: tarballs
+          path: tarballs
+    - name: Extract tarball
+      run: tar xzf tarballs/*.tar.gz -C $RUNNER_TEMP
+    - name: update Github_env
+      run: export TAR_DIR="$RUNNER_TEMP/`basename tarballs/*.tar.gz .tar.gz`" &&
+          echo TAR_DIR="$TAR_DIR" >>$GITHUB_ENV
+          
+# build config vars
+    - name: build CFLAGS and CXXFLAGS variables
+      run: echo CFLAGS_var="-Wall -pedantic -O0" >> $GITHUB_ENV
+           && echo CXXFLAGS_var="-Wall -pedantic -O0" >> $GITHUB_ENV
+      # Note: We want to use '-Werror', but if we already provide it at the configure step
+      #       we get errors in configure that we cannot remove (for example the autotools way
+      #       of checking for libm results in a compiler warning).
+      #       Thus, we add '-Werror' later at the make step.
+    - name: less-test-option
+      if: github.event_name == 'pull_request'
+      run: export LESS_TEST_OPTION="--enable-less-tests"
+            && echo LESS_TEST_OPTION="$LESS_TEST_OPTION" >> $GITHUB_ENV
+    - name: build config variables
+      run: export CONFIG_OPTIONS="--without-blas ${LESS_TEST_OPTION}"
+            && export CONFIG_DEBUG="$CONFIG_OPTIONS --enable-debug --enable-mpi"
+            && echo CONFIG_OPTIONS="$CONFIG_OPTIONS" >> $GITHUB_ENV
+            && echo CONFIG_DEBUG="$CONFIG_DEBUG" >> $GITHUB_ENV
+    - name: Check vars
+      run: echo "[$CONFIG_DEBUG]"
+    - name: configure from Tarball
+      run: mkdir build_tar && cd build_tar && $TAR_DIR/configure $CONFIG_DEBUG CFLAGS="$CFLAGS_var" CXXFLAGS="$CXXFLAGS_var"
```

Comment:
We are still in the t8code git directory, not in the tarball directory. We are currently not building from the tarball

Reply:
No, `$TAR_DIR` points to the tarball directory.